packages/drivers/detox/index.ts

```diff
@@ -3,19 +3,47 @@ import {
   TestingFrameworkDriver,
   TestingFrameworkDriverConfig,
 } from "@wix-pilot/core";
-import detox from "detox";
-import { expect } from "@jest/globals";
 import * as fs from "fs";
 import * as path from "path";
 
+import { expect as jestExpect } from "expect";
+
+/**
+ * Pilot driver for the Detox testing framework.
+ */
 export class DetoxFrameworkDriver implements TestingFrameworkDriver {
+  private _detox: any;
+
+  /**
+   * Creates a new Detox driver instance.
+   * @param detox optional Detox instance to use, if not provided, it will be imported from the
+   *  `detox` package (peer dependency). The reason for this parameter is to allow usage from the Detox package itself.
+   */
+  constructor(detox?: any) {
+    this._detox = detox;
```

Comment:
providing `detox` is optional. added for using the driver from Detox itself since it can't require `detox` when being called from the `detox` package.